//multiple deta exception
int main(){
    int x;
    cout<<"enter the value of x\n";
    cin>>x;
    try
    {
        if(x==0)
        throw 1;
        else if(x<0)
        throw 1.2f;
        else if(x>0)
        throw 'g';
    }
    catch(int x)
    {cout<<"number is zero\n";
    }
    catch(float x)
    {cout<<"number is negative\n";
    }
    catch(char ch)
    {
       cout<<"number is positive\n"; 
    }
    
}
